src/engine/SCons/Tool/install.xml

```diff
@@ -50,9 +50,23 @@ a builder.
 </para>
 
 <example_commands>
-env.Install('/usr/local/bin', source = ['foo', 'bar'])
+env.Install(target='/usr/local/bin', source=['foo', 'bar'])
```

Comment:
target's not required, nor is source.Perhaps we should list the allowed variations with and without keyword(s)?

Reply:
sure.  I was trying to nudge people in the direction of slightly more readable scripts by showing the use of keyword args...